consensus/consensus-types/src/safety_data.rs

```diff
@@ -17,6 +17,8 @@ pub struct SafetyData {
     #[serde(default)]
     pub one_chain_round: u64,
     pub last_vote: Option<Vote>,
+    pub last_order_vote: Option<OrderVote>,
```

Comment:
Why do we need to keep this?

Reply:
Similar to the 2-chain safety rules, I'm checking if an order vote for the round has been sent before. If an order vote has been previously sent, the validator sends the same order vote again.

Reply:
it's not possible to equivocate order vote compared to normal vote because it carries the qc so we can just re-sign (I think)

Reply:
Removed.